tests/deploy_test.go

```diff
@@ -13,6 +14,88 @@ import (
 
 var emptyQueryString = ""
 
+func Test_Scaling_Up(t *testing.T) {
+
+	functionRequest := requests.CreateFunctionRequest{
+		Image:      "functions/alpine:latest",
+		Service:    "test-scaling",
+		Network:    "func_functions",
+		EnvProcess: "env",
+	}
+
+	deployStatus, deployErr := deploy(t, functionRequest)
+	if deployErr != nil {
+		t.Errorf(deployErr.Error())
+		t.Fail()
+		return
+	}
+
+	if deployStatus != http.StatusOK && deployStatus != http.StatusAccepted {
+		t.Errorf("got %d, wanted %d or %d", deployStatus, http.StatusOK, http.StatusAccepted)
+	}
+
+	invoke(t, functionRequest.Service, emptyQueryString, http.StatusOK)
+
+	err := scale(t, functionRequest.Service, "10")
+	if err != nil {
+		t.Errorf(err.Error())
+		t.Fail()
+		return
+	}
+
+	replicas, err := getReplicas(t, functionRequest.Service)
+	if err != nil {
+		t.Errorf(err.Error())
+		t.Fail()
+		return
+	}
+
+	want := "10"
+	if want != fmt.Sprintf("%d", replicas) {
+		t.Errorf("Want %s replicas, Got %d", want, replicas)
+	}
+
+}
+
+func scale(t *testing.T, fnName string, replicas string) error {
```

Comment:
It'd be better not to pass testing.T and then deal with any testing stuff in the caller.

Reply:
Fixed in PR#24.

I noticed that other functions have testing.T parameters. Examples in deploy_test.go are
* line 267, deploy()
* 283, list()
* 308, get()

That might explain why @ivanayov did something similar.